src/options/index.ts

```diff
@@ -67,12 +67,28 @@ urlForm.addEventListener("submit", (e) => {
     return alert("You can't add a website while the focus mode is running.");
   if (!urlInput.value) return alert("Please enter a URL.");
 
-  blockList.push(urlInput.value);
-  browser.storage.local.set({ blockList });
-  addUrlListElement(urlInput.value);
+  if (!isValidURL(urlInput.value)) return alert("Please enter a valid URL.") // verify URL
+
+  if (checkUrlListElements(urlInput.value)) {
+      // if url exists in website-list div, return alert
+      return alert("This URL already exists")
+  } else {
+    // if url doesnt exist in website-list div, add new element to
```

Comment:
You can remove all your commentaries, it's easy to understand what your code does.